### PR TITLE
set up and first end point completed

### PR DESCRIPTION
**Pull request by FreddieStoker7**

### CODE REVIEW

**Review by popatre — CHANGES_REQUESTED** [review]
I think you're only adding your test files again when pushing. 
Make sure you're in the root of your project when using git add .

Also, this one test i can see, doesnt actually test anything, it just console logs the topics.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+const app = express();
+
+app.get('/api/topics', getTopics)
+app.get('/api/articles/article_id', getArticle)
+
+
+
+
+
+app.all('/*', (req, res, next) => {
```
Comment (DCrawley94) [inline app.js:13]: nice!

File: controllers/app-controllers.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const { selectTopics } = require("../models/app-models.js");
+
+exports.getTopics = (req, res, next) => {
+  selectTopics().then((topics) => {
+    res.status(200).send({ topics: topics });
```
Comment (DCrawley94) [inline controllers/app-controllers.js:5]: This is a valid way of sending the response however if you wanted to be extra fancy you could use this syntax:
`res.status(200).send({topics})`

File: models/app-models.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection.js");
+
+exports.selectTopics = async () => {
+  const topics = await db.query("SELECT * FROM topics;");
+    return topics.rows;
+};
+
+exports.selectArticle = async (article_id) => {
```
Comment (DCrawley94) [inline models/app-models.js:8]: Make sure you write your tests first before writing the endpoint. Your code should be written according to what is expected by the tests :)

**Review by DCrawley94 — COMMENTED** [review]
Looking good!

**Pull request merged**